src/Corporation/Actions.ts

```diff
@@ -90,6 +91,27 @@ export function IssueDividends(corporation: Corporation, rate: number): void {
   corporation.dividendRate = rate;
 }
 
+export function IssueNewShares(corporation: Corporation, amount: number, max: number): number {
+  if (isNaN(amount) || amount < 0 || amount > max) {
```

Comment:
Doesn't issuing new shares also have to be in increments of 10m? That doesn't seem to be enforced here.

Reply:
aw shucks. I calculated the 10m increment amount for the max new shares only, you're absolutely right